AsyncDisplayKit/Private/ASDisplayNodeLayout.h

```diff
@@ -0,0 +1,58 @@
+//
+//  ASDisplayNodeLayout.h
+//  AsyncDisplayKit
+//
+//  Created by Michael Schneider on 08/26/16.
+//
+//  Copyright (c) 2014-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under the BSD-style license found in the
+//  LICENSE file in the root directory of this source tree. An additional grant
+//  of patent rights can be found in the PATENTS file in the same directory.
+//
+
+#pragma once
+
+#import "ASDimension.h"
+
+@class ASLayout;
+
+/*
+ * Represents a connection between an ASLayout and a ASDisplayNode
+ * ASDisplayNode uses this to store additional information that are necessary besides the layout
+ */
+struct ASDisplayNodeLayout {
```

Comment:
The name of this struct is not very clear. This struct stores a layout and additional information related to it. Maybe call it `ASLayoutBundle` or `ASLayoutContext`?